### PR TITLE
[Add] New Rule: css property order #8

### PR DESCRIPTION
**Pull request by koozaki**
hi, @pirosikick. I dealt issue/#8.

Please check (include design).
I was troubled whether to separete the config ( src/config/css-property-order-list.js ).

### CODE REVIEW

File: src/rules/css-property-order.js [hunk 1]
```diff
@@ -0,0 +1,49 @@
+'use strict';
+
+import cssPropertyOrderList from '../config/css-property-order-list';
+
+const css_property_order_list = cssPropertyOrderList();
```
Comment (pirosikick) [inline src/rules/css-property-order.js]: Please use lower camel case.

File: src/rules/css-property-order.js [hunk 2]
```diff
@@ -0,0 +1,49 @@
...
+  return {
+
+    CallExpression(node) {
+
+      let {callee, arguments: args} = node;
+
+      if (!callee.property || callee.property.name !== 'css' || args.length !== 1 || args[0].type !== 'ObjectExpression' || !args[0].properties[0]) return;
+
+      let prev_order = -2,
+        key_list = [];
```
Comment (pirosikick) [inline src/rules/css-property-order.js]: Please use lower camel case too.

File: src/config/css-property-order-list.js [hunk 3]
```diff
@@ -0,0 +1,33 @@
...
+  'font',
+  'text-decoration',
+  'text-align',
+  'vertical-align',
+  'white-space',
+  // 'other text'
+  'content'
+];
+
+export default function() {
```
Comment (pirosikick) [inline src/config/css-property-order-list.js]: Why do you use the function to return the property list rather than exporting the list directly?

Comment (koozaki) [inline src/config/css-property-order-list.js]: oh... That's so true.

File: src/rules/css-property-order.js (hunk 2 above)
Comment (koozaki) [inline src/rules/css-property-order.js]: :ok:

File: src/rules/css-property-order.js (hunk 1 above)
Comment (koozaki) [inline src/rules/css-property-order.js]: :ok:

**Comment by pirosikick** [comment]
@koozaki Thanks!

**Pull request merged**